frontend/packages/kubevirt-plugin/src/components/vm-disks/vm-disks.tsx

```diff
@@ -48,68 +48,65 @@ const getStoragesData = ({
     diskInterface: disk.getDiskInterface(),
     size: disk.getReadableSize(),
     storageClass: disk.getStorageClassName(),
+    metadata: { name: disk.getName(), type: disk.getType() },
```

Comment:
does the type need to be wrapped in metadata?

Reply:
nice, moving :-)